### DISCUSSION TITLE
Questions about establishing the Martini3 force field top file for thiophene ring structures.

### DISCUSSION THREAD

**Discussion by zhangtrzhangtr** [ricalessandri/Martini3-small-molecules#44] [Q&A] [answered]
![1710314141641](https://github.com/ricalessandri/Martini3-small-molecules/assets/161429244/2069b0d8-6841-4de6-95b3-0a6c4f52933c)
Why is the bond connecting two adjacent thiophene rings formed using the virtual sites at the centers of the two thiophene rings, rather than connecting the beads on the rings or connecting a virtual site to a bead on the ring? For example, defining the bond length between two thiophene rings using bead 3 and bead 5 shown in the figure, or connecting the virtual site 4 with bead 5 to control the bond length between the two thiophene rings. I am asking this because, when I was creating a bond between two slightly more complex ring structures, I couldn't use the virtual sites on the rings to control the bond length between the two ring structures. When I connected the two virtual sites, the two ring structures would separate during the coarse-grained molecular dynamics simulation even though I had excluded the non-bonded interactions between the beads on the two rings. I could only control the bond length by connecting the actual beads on the rings. Thank you for your guidance.

**Comment by ricalessandri** [comment, marked as answer]
@zhangtrzhangtr Sorry for the late reply. Background on the choice of this constructions (which dates back to [this paper](https://pubs.acs.org/doi/full/10.1021/jacs.6b11717) and [this PhD thesis](https://repository.tudelft.nl/islandora/object/uuid:d2b30d39-3883-40d5-b283-033053d2f62f)) are explained in quite some detail in the [M3 small molecule paper](https://doi.org/10.1002/adts.202100391). It is what I called "divide and conquer" strategy.

In brief, it is a convenient construction for a number of reasons, but mainly for the dihedral definition (it will be numerically stable given the ~90 degree internal angles) and relative simplicity. However, it's not a *must*. If you find that something else works better for your model, you should go with that.

Please have a look at the ["divide and conquer" strategy in the M3 small molecule paper](https://doi.org/10.1002/adts.202100391) and feel free to follow up.
